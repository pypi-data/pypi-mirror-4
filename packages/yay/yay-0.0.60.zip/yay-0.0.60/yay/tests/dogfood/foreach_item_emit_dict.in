foolist:
  - name: john
    age: 25
  - name: simon
    age: 41
  - name: sandra
    age: 47
  - name: alice
    age: 18
  - name: bardrone
    age: 3

bar:
   maxage: 40

baz.foreach person in foolist[@.age < bar.maxage]:
  nameage: ${person.name}${person.age}
  agename: ${person.age}${person.name}

