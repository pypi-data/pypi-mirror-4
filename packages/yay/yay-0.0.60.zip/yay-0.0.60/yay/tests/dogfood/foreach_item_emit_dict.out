foolist:
  - name: john
    age: 25
  - name: simon
    age: 41
  - name: sandra
    age: 47
  - name: alice
    age: 18
  - name: bardrone
    age: 3

bar:
   maxage: 40

baz:
  - nameage: john25
    agename: 25john
  - nameage: alice18
    agename: 18alice
  - nameage: bardrone3
    agename: 3bardrone

